### PR TITLE
Added deep link in globalsettings.

### PR DESCRIPTION
**Pull request by chesa10**

### CODE REVIEW

File: locales.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
         "PluginDisplayname": "Slack",
         "InvalidSlackConfiguration": "Invalid Slack Configuration",
         "SlackAppOAuthToken": "Slack App OAuth Token",
+        "EnableMobileDeepLink": "Enable Mobile Deep Link",
```
Comment (dkroukamp) [inline locales.json]: Set the string to rather be  `Enable Mobile App Freindly Links`

File: manifest.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "request.aspx"
   ],
   "GlobalSettings": [
-    "@@SlackAppOAuthToken"
+    "@@SlackAppOAuthToken",
+	"@@EnableMobileDeepLink"
```
Comment (dkroukamp) [inline manifest.json]: call this `EnableMobileLink`

File: handler/ApiHandler.ashx [hunk 3]
```diff
@@ -64,22 +96,160 @@ public class ApiHandler : PluginHandler
...
+                    }
+                },
+            }
+        };
+    }
+
     public class SlackClient
     {
-        private readonly Uri uri;
+        private readonly Uri Base;
```
Comment (Kgothatso09) [inline handler/ApiHandler.ashx]: Change the Base to base.

File: manifest.json (hunk 2 above)
Comment (Kgothatso09) [inline manifest.json]: Please check indentation

**Pull request merged**